.github/workflows/Nightly.yml

```diff
@@ -58,6 +59,10 @@ jobs:
         run: |
           git push -u origin HEAD
 
+      - name: Wait for push to complete
+        if: steps.vendor.outputs.vendor != '' && github.event_name != 'pull_request'
+        run: sleep 10  # Give GitHub a moment to process the push
```

Comment:
Could you expand on why this sleep is necessary? Doesn't the push complete otherwise?

Reply:
M, I've just copied this bit from the latest ODBC workflow to keep JDBC in sync with it. I will re-check.

Reply:
As an update on this, I've got some input on GH workflows from Carlo, as a result the vendoring process in ODBC is [being changed](https://github.com/duckdb/duckdb-odbc/pull/96) to import engine sources into a separate branch, create a PR on this branch and merge this PR automatically. In this variant the testing workflow is dispatched by triggering the `pull_request` event instead of a direct dispatch, so we do not need to wait manually until the push completes. I intend to bring the same workflow changes to JDBC once they are approved for ODBC, keeping this PR as a "draft" for now.

Reply:
Removed the timeout, now using `push` event instead that is expected only to be triggered by Github after the push is actually complete. Also removed the periodic trigger and submitted duckdb/duckdb/pull#17099 on the main DuckDB repo side to notify the JDBC vendoring process.